### ISSUE TITLE
Release a new version already!

### ISSUE THREAD

**Issue by mitchelljustin** [ryanmcgrath/twython#528]
We've had to use `git+https://github.com/ryanmcgrath/twython@36fda7ac02a8047d11ad4de12161a6ffd1e869ea` instead of `twython==3.7.0` in our requirements because on 3.7.0, fetching Direct Messages is broken. This causes `pip install` to reinstall it every time from scratch instead of using the cached package.

When is the next release slated?

**Comment by ryanmcgrath** [comment]
Yeah, see, I'm not going to jump in and fix this on the spot with that kind of tone - it's just opening Pandora's Box. You don't have to be nice or whatever, but you can't word it like that. This issue could've easily been titled "Can't cache package installs due to outdated version".

_Nobody_ is paid to work on this, and the people who have worked on it in the past have lives they're living. Shit happens, people get busy, or their use cases move on, etc. It's a free and open source library that, surprisingly, has found widespread use for over a decade in the face of an actively hostile API set.

There have been _multiple_ calls to enable other people to step up and provide stewardship/maintenance/etc. If things aren't happening, it's because those calls haven't been answered. Opening an issue with an exasperated title like yours will do nothing to move it along.

*tl;dr*: _If_ I find time later this week I'll bump it myself. _Don't_ disrespect the people who _have_ poured work into the project (and I'm speaking about other contributors more than myself here). GitHub and open source culture provide an illusion that free work is owed when that simply isn't the case.

**Comment by ryanmcgrath** [comment]
`3.8.0` is out.

**Comment by mitchelljustin** [comment]
Hey @ryanmcgrath, I'm sorry for being rude. You're absolutely right, this is an open-source passion project and I had no right to make demands like I did. 

Thanks a lot for releasing the new version anyway. I'll be more respectful towards you and other OSS devs in the future.

**Comment by ryanmcgrath** [comment]
Thanks for the apology. We're good. :)
